docs/superpowers/plans/2026-08-06-adfa-4510-codeaction-tooltips.md

```
@@ -0,0 +1,878 @@
+```
+
+- [ ] **Step 3: Run the test to verify it fails**
+
+```bash
+flox activate -d flox/local -- ./gradlew :actions:testV8DebugUnitTest \
+  --tests "com.itsaky.androidide.actions.ActionTooltipResolutionTest"
+```
+
+Expected: FAIL. Two distinct failures:
+- a compile error, `Unresolved reference: findAction` (the `Int` overload does not exist yet)
+- once that compiles, `retrieveTooltipTag reads an action that overrides only the property` fails with `expected: editor.codeactions.comment but was: ` (empty)
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)